### DISCUSSION TITLE
Py5 on RosettaCode

### DISCUSSION THREAD

**Discussion by vsquared** [py5coding/py5generator#567] [General]
Does it make any sense to add Py5 as a language on https://rosettacode.org/wiki/Rosetta_Code

**Comment by villares** [comment]
That's a great question!

I think py5 examples might go under Python now... I used to contribute with Processing Python Mode examples for RosettaCode, Jeremy Douglass had a repository of Processing & related things from RosettaCode...  https://github.com/jeremydouglass/rosetta_examples_p5/tree/master/examples/Python

On a side note, we had this old discussion here: https://github.com/py5coding/py5generator/discussions/104
and I have some other examples at: https://github.com/villares/py5examples/tree/processing-python-mode-examples/examples-from-Processing-Python-mode
I wish I had more time work on those...

**Comment by hx2A** [comment]
Interesting idea, @vsquared !

I just looked at the website and on this page:

https://rosettacode.org/wiki/Category:Programming_Languages?subcatfrom=OPENEDGE%2FPROGRESS%0AOpenEdge%2FProgress#mw-subcategories

I see that Processing, Processing.py and Processing.R are included. I guess if Processing.py and Processing are there, I can't think of any reason why py5 can't be there also. I do note that P5 is not included. Perhaps those 3 were added long ago by someone (Jeremy?) who was mainly interested in Processing, Processing.py and Processing.R? I vaguely recall Jeremy is the author of Processing.R?

That reminds me, I haven't talked to Jeremy in ages. I hope he is doing well.

**Reply by villares** [reply]
Jeremy is wonderful, I miss interacting with him in the forum. I think (I might be wrong) that he was not the creator of Processing.R but became the maintainer of it. I tried very hard to convince lot's of R people I met to have a look at it...

**Reply by hx2A** [reply]
I wonder what happened to him? I just checked his Github page and his activity is zero for the past year.
